modules/tinymce/src/plugins/table/test/ts/browser/ClipboardTest.ts

```diff
@@ -1,38 +1,43 @@
-import { Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { LegacyUnit, TinyLoader } from '@ephox/mcagar';
+import { describe, it } from '@ephox/bedrock-client';
+import { LegacyUnit, TinyHooks } from '@ephox/mcagar';
+import { assert } from 'chai';
 
 import Editor from 'tinymce/core/api/Editor';
 import Tools from 'tinymce/core/api/util/Tools';
 import Plugin from 'tinymce/plugins/table/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.table.ClipboardTest', (success, failure) => {
-  const suite = LegacyUnit.createSuite<Editor>();
-
-  SilverTheme();
-  Plugin();
+describe('browser.tinymce.plugins.table.ClipboardTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'table',
+    indent: false,
+    valid_styles: {
+      '*': 'width,height,vertical-align,text-align,float,border-color,background-color,border,padding,border-spacing,border-collapse'
+    },
+    theme: 'silver',
```

Comment:
Not sure if it matters but I think this should be removed?

Reply:
Yes it should, thanks!